calcom/cal.com
# fix: Fixed Issue #10291 removed brand color for tooltip

## What does this PR do?
fixes #10291 

The issue simply happened because we have given bg-brand-default as background color of tooltip, which gets bg color from brand.
I had removed the class and gave bg-inverted static class which gives background color irrespective of brand color


## Type of change


## How should this be tested?
See below screenshots taken after fix
https://drive.google.com/file/d/17sF5XTvlXA9rGiUXjAVx3Zog3sZfb8nW/view?usp=sharing
https://drive.google.com/file/d/1rPYfeSwav58OtLifYEpijcy69ZKXJGTD/view?usp=sharing


## Mandatory Tasks
